tools/OresToFieldGuide/JsonUtility.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OresToFieldGuide
+{
+    public class JsonPropertyConverter : JsonConverter
```

Comment:
what does this do that newtonsoft doesn't already do?

Reply:
This was a leftover from me trying to fix an issue, deleted it